source/gui/settingsDialogs.py

```diff
@@ -3040,11 +3048,26 @@ def makeSettings(self, settingsSizer):
 			label=_("&Remove")
 		).Bind(wx.EVT_BUTTON, self.OnRemoveClick)
 
+		if(self._profile.name):
+			bHelper.addButton(
+				parent=self,
+				# Translators: The label for the import entries from default profile dictionary
+				label=_("&import entries from default profile dictionary")

```

Comment:
Please use uppercase for first letter as for the other buttons.
```suggestion
				label=_("&Import entries from default profile dictionary")

```